dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/pom.xml

```diff
@@ -141,5 +141,6 @@
             <groupId>com.zaxxer</groupId>
             <artifactId>HikariCP</artifactId>
         </dependency>
+
```

Comment:
Please remove unnessnary change.

Reply:
Ok, I'll remove the unnecessary changes right away